enterprise/templatetags/enterprise.py

```diff
@@ -0,0 +1,30 @@
+"""
+Template tags for enterprise apps.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django import template
+from django.utils.safestring import mark_safe
+
+
+register = template.Library()  # pylint: disable=invalid-name
+
+
+@register.simple_tag(name="fa_icon")
+def fa_icon(tags):
+    """
+    Django template tag that returns font awesome icon depending upon message tags.
+
+    Usage:
+        {% fa_icon "success" %}
+    """
+    if 'success' in tags:
```

Comment:
django message `tags` are space separated string list and can include multiple tags. So I think we can improve this template tag by returning corresponding number of font awesome icons. e.g, `"info success"` can be returned as  `mark_safe('<i class="fa fa-info-circle" aria-hidden="true"></i> <i class="fa fa-check-circle" aria-hidden="true"></i>')`.